### PR TITLE
Improvements to load balacer topology

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Signed-off-by: Prajyot Parab <Prajyot.Parab1@ibm.com>

### CODE REVIEW

File: modules/7_ibmcloud/load_balancer.tf [hunk 1]
```diff
@@ -18,31 +18,44 @@
...
   apps_servers_count = var.worker_count == 0 ? var.master_count : var.worker_count
 }
 
-resource "ibm_is_lb" "load_balancer" {
-  name            = "${var.name_prefix}-loadbalancer"
+resource "ibm_is_lb" "load_balancer_internal" {
+  name            = "${var.name_prefix}-internal-loadbalancer"
+  subnets         = [var.vpc_subnet_id]
+  security_groups = [ibm_is_security_group.ocp_security_group.id]
+  type            = "private"
```
Comment (bpradipt) [inline modules/7_ibmcloud/load_balancer.tf:37]: should we add `tags` field here pointing to ${var.name_prefix} ?

**Review by bpradipt — APPROVED** [review]
/lgtm
Only question is do we need to start adding `tags` to the resources ?

**Comment by yussufsh** [comment]
> Only question is do we need to start adding `tags` to the resources ?

Tags are available for VPC/Classic TF resources, we should use them.

**Review by yussufsh — APPROVED** [review]
/lgtm
Would suggest proper documentation on the architecture for external services including the two load balancers. @bpradipt

**Pull request merged**